vuepress/docs/v6.3.2/tutorials/micro-frontends/react.md

```diff
@@ -235,7 +234,7 @@ npm run build
 | build/static/css/main.5f361e03.chunk.css  | `static/css/main.css`     | Stylesheet
 
 ::: warning Generated Build Files
-We rename the JavaScript and CSS files so we can deploy new versions of the micro frontend without having to update the `Custom UI` field of our widget to reference the new files.
+The JavaScript and CSS files are renamed so it can deploy the new versions of the micro frontend without having to update the `Custom UI` field of the widget.
```

Comment:
Technically the user is the one doing the deploy in this tutorial - maybe "it can deploy" should be "we can manually deploy"

Reply:
i would do "...are renamed to deploy..." ;)

Reply:
The JavaScript and CSS files are renamed to manually deploy the new versions of the micro frontend without having to update the `Custom UI` field of the widget.